package.json

```
@@ -18,7 +21,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "lint": "eslint './src/**/*.{js,jsx}'",
+    "lint": "eslint  './src/**/*.{js,jsx}'",
```

Лучше оставь один пробел, как на мастере :D
```suggestion
    "lint": "eslint './src/**/*.{js,jsx}'",
```